Gopkg.toml

```diff
@@ -0,0 +1,48 @@
+
+# Gopkg.toml example
+#
+# Refer to https://github.com/golang/dep/blob/master/docs/Gopkg.toml.md
+# for detailed Gopkg.toml documentation.
+#
+# required = ["github.com/user/thing/cmd/thing"]
+# ignored = ["github.com/user/project/pkgX", "bitbucket.org/user/project/pkgA/pkgY"]
+#
+# [[constraint]]
+#   name = "github.com/user/project"
+#   version = "1.0.0"
+#
+# [[constraint]]
+#   name = "github.com/user/project2"
+#   branch = "dev"
+#   source = "github.com/myfork/project2"
+#
+# [[override]]
+#  name = "github.com/x/y"
+#  version = "2.4.0"
+
+
+[[constraint]]
+  name = "github.com/BurntSushi/toml"
+  version = "0.2.0"
+
+[[constraint]]
+  name = "github.com/RobotsAndPencils/buford"
+  version = "0.12.0"
+
+[[constraint]]
+  name = "github.com/client9/reopen"
```

Comment:
It's good to write branch, version or revision to omit following warning from latest dep.

```
dep: WARNING: branch, version, revision, or source should be provided for "github.com/client9/reopen"
```

Reply:
I'll update them to include revisions. Thanks for the advise.